src/tools/ScenarioMeasurement/Startup/ProcessTimeParser.cs

```diff
@@ -70,9 +69,7 @@ public IList<Counter> Parse(string mergeTraceFile, string processName)
 
                 source.Kernel.ProcessEndGroup += evt =>
                 {
-                    // not using processname here because it's not tracked by the state machine when the process just ended
-                    // but ImageFileName exists
-                    if (pid.HasValue && evt.ProcessID == pid && evt.ImageFileName == imageFileName)
+                    if (pid.HasValue && evt.ProcessID == pid && commandLine.Equals(evt.CommandLine.Replace("\"", ""), StringComparison.OrdinalIgnoreCase))
```

Comment:
I know we had trouble with the processName field on end events. Is CommandLine valid in this case?

Reply:
Yes evt.CommandLine does not have the same issue with process name